zetasql/fuzzing/protobuf/parameter_grammar.proto

```diff
@@ -31,4 +33,45 @@ message Whitespace {
 
 message Default {
     required bytes content = 1;
+}
+
+message Identifier {
+    enum Type {
+        COLUMN = 0;
+        PARAMETER = 1;
+    }
+    required bytes name = 1;
+    required Type type = 2;
+}
+
+message Value {
+    required Literal literal = 1;
+    optional Identifier as_variable = 2;
+}
+
+message Literal {
+    oneof literal_oneof {
+        zetasql.TypeKind null_literal = 1;
+        bool bool_literal = 2;
+        string string_literal = 3;
+        bytes bytes_literal = 4;
+        IntegerLiteral integer_literal = 5;
+        NumericLiteral numeric_literal = 6;
+    }
+    required Default default_value = 7;
+}
+
+message IntegerLiteral {
+    oneof integer_oneof {
+        int32 int32_literal = 1;
+        uint32 uint32_literal = 2;
+        int64 int64_literal = 3;
+        uint64 uint64_literal = 4;
+    }
+    required Default default_value = 5;
+}
+
+// TODO: Support high precision value representation
+message NumericLiteral {
+    required bytes value = 1;
```

Comment:
You may be able to alternatively get away with a series of int64s here instead of bytes, which would fix the problem with the bytes winding up being an invalid numeric value.

Reply:
It does make literal conversion harder in that case, and we have to use zetasql::NumericValue to help us convert them to numeric literal. I would assume this is less ideal as we now are rely on components that we want to test, and can also limit our test input space (since the output is constrained by whatever the helper function from zetasql::NumericValue)

Reply:
That makes sense.